costajohnt/w03-d01homework.txt

```diff
@@ -0,0 +1,13 @@
+Get all icecream flavors “/icecream”
+
+Get icecream by ID “/icecream/:id”
+
+Get icecream fruity “/fruity/:id/icecream”
+
+Get ice-cream gluten free “/glutenfree/:id/icecream”
+
+Get ice-cream chunky “/chunky/:id/icecream”
+
+Post icecream “/icecream/:id”
```

Comment:
normally we won't specify an id for post - its creating a new one and the user doesn't get to pick the ID (usually)